mmv1/products/cgc/terraform.yaml

```diff
@@ -206,6 +206,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_public_ip"
+        primary_resource_type: "google_sql_database_instance"
+        primary_resource_id: "sqlserver_public_ip_instance_name"
+        vars:
+          sqlserver_public_ip_instance_name: "sqlserver-public-ip-instance-name"
+          deletion_protection: "true"
+        test_vars_overrides:
+          deletion_protection: "false"
+        ignore_read_extra:
```

Comment:
Probably need to add `- "root_password"` to `ignore_read_extra:` to pass the test